ts/private/ts_project.bzl

```diff
@@ -269,6 +286,7 @@ This is an error because Bazel does not run actions unless their outputs are nee
             ),
             env = {
                 "BAZEL_BINDIR": ctx.bin_dir.path,
+                "JS_BINARY__STDOUT_OUTPUT_FILE": stdout_file,
```

Comment:
nit: I'd feel safer if we didn't pass an environment variable with this key at all, rather than rely on an empty string value being interpreted the same as absence